playbooks/openshift/roles/common/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: reload firewalld
```

Comment:
That seems strange I would expect if you have a handler in the nginx role that you can call it from there even if there are other roles run first that it depends on.  I would have to setup an example to verify but  I believe we did it that way in class and it worked as expected.